src/azure-cli/azure/cli/command_modules/appservice/_help.py

```diff
@@ -1542,3 +1542,1398 @@
     type: command
     short-summary: Creates a remote connection using a tcp tunnel to your web app
 """
+
+helps['functionapp hybrid-connection'] = """
+    type: group
+    short-summary: commands that list, add, or remove hybrid connections from a sepcific functionapp
+    examples:
+        - name: list the hybrid connections
+          text: >
+              az functionapp hybrid-connection list -g [myResourceGroup] -n [myAppName]
+        - name: add a hybrid connection to the functionapp
+          text: >
+              az functionapp hybrid-connection add -g [myResourceGroup] -n [myAppName]
+              --namespace [hybridConnectionNamespaceName] --hybrid-connection [hybridConnectionArmUri]
+        - name: remove the hybrid connection from the functionapp
+          text: >
+              az functionapp hybrid-connection remove -g [myResourceGroup] -n [myAppName]
+              --namespace [hybridConnectionNamespace] --hybrid-connection [hybridConnectionName]
+"""
+
+helps['functionapp hybrid-connection list'] = """
+    type: command
+    short-summary: lists all the hybrid connections set on the specified functionapp
+    examples:
+        - name: list the hybrid connections
+          text: >
+              az functionapp hybrid-connection list -g [myResourceGroup] -n [myAppName]
+"""
+
+helps['functionapp hybrid-connection add'] = """
+    type: command
+    short-summary: adds an existing hybrid connection to the specified functionapp
+    parameters:
+        - name: --hybrid-connection
+          type: string
+          short-summary: 'Hybrid connection name'
+        - name: --namespace
+          type: string
+          short-summary: 'Hybrid connection namespace'
+    examples:
+        - name: add a hybrid connection to the app
+          text: >
+            az functionapp hybrid-connection add -g [myResourceGroup] -n [myAppName]
+            --namespace [hybridConnectionNamespaceName] --hybrid-connection [hybridConnectionArmUri]
+"""
+
+helps['functionapp hybrid-connection remove'] = """
+    type: command
+    short-summary: disconnects the specified hybrid connection from the specified functionapp
+    parameters:
+        - name: --hybrid-connection
+          type: string
+          short-summary: 'hybrid connection Arm Uri'
+        - name: --namespace
+          type: string
+          short-summary: 'Hybrid connection namespace'
+    examples:
+        - name: remove the hybrid connection from the functionapp
+          text: >
+            az functionapp hybrid-connection remove -g [myResourceGroup] -n [myAppName]
+            --namespace [hybridConnectionNamespace] --hybrid-connection [hybridConnectionName]
+"""
+
+helps['webapp hybrid-connection'] = """
```

Comment:
Same comment here as well. This is just the group so keep this limited to the group description